src/commands/dev.ts

```diff
@@ -250,7 +263,21 @@ export async function command(commandOptions: CommandOptions) {
         }
         return `/web_modules/${spec}.js`;
       });
+
+      if (!fileLoc.includes('web_modules') && !fileLoc.includes('node_modules')) {
+        const imports = await scanCodeImportsExports(builtFileResult.result);
+        for (const imp of imports.reverse()) {
```

Comment:
you shouldn't need to reverse here, we only do that when we're editing the file itself